tests/Fieldtypes/LinkTest.php

```diff
@@ -3,43 +3,75 @@
 namespace Tests\Fieldtypes;
 
 use Facades\Statamic\Routing\ResolveRedirect;
+use Mockery;
 use Statamic\Entries\Entry;
+use Statamic\Fields\ArrayableString;
 use Statamic\Fields\Field;
 use Statamic\Fieldtypes\Link;
+use Tests\PreventSavingStacheItemsToDisk;
 use Tests\TestCase;
 
 class LinkTest extends TestCase
 {
+    use PreventSavingStacheItemsToDisk;
```

Comment:
@jasonvarga don't think this is needed now, as the entries are mocked

Reply:
Totes 👍